packages/web-components/fast-components/src/menu-item/menu-item.styles.ts

```diff
@@ -125,15 +144,19 @@ export const menuItemStyles = (context, definition) =>
         fill: ${neutralForegroundRest};
     }
 
-    :host([aria-haspopup="menu"]),
-    :host([role="menuitemcheckbox"]),
-    :host([role="menuitemradio"]) {
+    :host(:not(.indent-2)[aria-haspopup="menu"]),
+    :host(:not(.indent-2)[role="menuitemcheckbox"]),
+    :host(:not(.indent-2)[role="menuitemradio"]) {
```

Comment:
`:not()` can get fairly expensive - are there potential alternatives here which might be more performant?

Reply:
Was able to remove most of the nots.